Add glossaries working group (#58)
thegooddocsproject/website-hugo: content/en/working-group/glossaries.md

---
title: Glossaries
poc: "[Cameron Shorter](https://thegooddocs.slack.com/team/UKTGLQNGG)"

> meeting: Monthly on Tuesdays at 7AM UTC(on Google Meet)

Reviewer: ```suggestion
meeting: Monthly on third Tuesdays of the month, at 7AM UTC (on Google Meet)
```

status: Active
description: Creating cross-organizational and inter-operable glossaries with a standardised format for terms and definitions.
slackName: glossaries
slackID: C019A1EQJMC

> notesURL: https://docs.google.com/document/d/1Fjrl34ErnYammel9WmvXJ3rMWFANjoSiiGyyNSYOXUg/

Reviewer: ```suggestion
notesURL: https://docs.google.com/document/d/1yzncnQB7uYktPnAQ4DqZYVPDdYnqDrWmqHzcpXrjkPg/
```
I think this is supposed to reference our meeting notes page, as per above.

members:
  - "[Ankita Tripathi](https://thegooddocs.slack.com/team/D016LHJPDSR)"
  - "[Cameron Shorter](https://thegooddocs.slack.com/team/UKTGLQNGG)"
  - "[Codrina Maria Ilie](https://thegooddocs.slack.com/team/D01A8SW6CBX)"
  - "[Reese W. Plews](https://thegooddocs.slack.com/archives/D019RSVPE69)"
  - "[Rob Atkinson](https://thegooddocs.slack.com/team/U01DYRWG43X)"

draft: false
---

The goal of our community is to create a template for glossaries that can be easily integrated by anyone creating glossaries for their own project. The pilot will build processes and tools for the generic use case, developing and applying them within the geospatial mapping domain.

> We plan to use the base template (a "template of templates") that we've created as the starting point to create a template for glossaries. We will also create a process for a central glossary owner to accept, triage, and refine proposals for new terms from their community.

Reviewer: ```suggestion
* We plan to use the base template (a "template of templates") that we've created as the starting point to create a template for glossaries.
* We will establish processes for sharing terms between organizations and glossaries.
* We will create a process for a central glossary owner to accept, triage, and refine proposals for new terms from their community.
```
Adding in the use case of sharing terms between organizations.

---
title: Glossaries
poc: "[Cameron Shorter](https://thegooddocs.slack.com/team/UKTGLQNGG)"
meeting: Monthly on third Tuesdays of the month, at 7AM UTC (on Google Meet)
status: Active
description: Creating cross-organizational and inter-operable glossaries with a standardised format for terms and definitions.
slackName: glossaries
slackID: C019A1EQJMC
notesURL: https://docs.google.com/document/d/1yzncnQB7uYktPnAQ4DqZYVPDdYnqDrWmqHzcpXrjkPg/
members:
  - "[Ankita Tripathi](https://thegooddocs.slack.com/team/D016LHJPDSR)"
  - "[Cameron Shorter](https://thegooddocs.slack.com/team/UKTGLQNGG)"
  - "[Codrina Maria Ilie](https://thegooddocs.slack.com/team/D01A8SW6CBX)"
  - "[Reese W. Plews](https://thegooddocs.slack.com/archives/D019RSVPE69)"
  - "[Rob Atkinson](https://thegooddocs.slack.com/team/U01DYRWG43X)"

draft: false
---

> The goal of our community is to create a template for glossaries that can be easily integrated by anyone creating glossaries for their own project. The pilot will build processes and tools for the generic use case, developing and applying them within the geospatial mapping domain.

Reviewer 2: ```suggestion
The goal of our community is to create a template for glossaries that can be easily integrated by anyone creating glossaries for their own project.
The pilot will build processes and tools for the generic use case, developing and applying them within the geospatial mapping domain.
```
